nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/connect/client/status/ConnectStatusHolder.java

```diff
@@ -54,9 +54,9 @@ protected ConnectRegistrationService getService() {
         return Framework.getLocalService(ConnectRegistrationService.class);
     }
 
-    public boolean isRegistred() {
+    public boolean isRegistered() {
```

Comment:
Yes, you're right. This is a public method, so adding the `@since` in important. Unfortunately, the PR has already been merged, so this will be done in the fix-up PR, see https://github.com/nuxeo/nuxeo/pull/891